test/runtests.jl

```diff
@@ -659,4 +659,33 @@ end
     @test RGB(Gray(0.2), Gray(0.3), Gray(0.4)) == RGB(0.2, 0.3, 0.4)
 end
 
+@testset "RGB565" begin
+    c = RGB565(31, 63, 31)  # white
```

Comment:
```suggestion
    c = RGB565(1, 1, 1)  # white
```

Isn't that much more clearly white? (If you prefer you could write this `RGB565(1.0, 1.0, 1.0)`.)